frontend/packages/kubevirt-plugin/src/selectors/vm-template/selectors.ts

```diff
@@ -26,6 +33,25 @@ export const getVMTemplate = (
     : undefined;
 };
 
+export const getVMCommonTemplate = (
```

Comment:
you should still find the template by name and namespace. If these name/namespace labels are missing than you can fallback to label/workload/os.

Reply:
added the regular check first